cmd/edit/addon/cmd.go

```diff
@@ -206,9 +206,13 @@ func run(cmd *cobra.Command, argv []string) {
 			if addOnInstallationParam != nil {
 				dflt = addOnInstallationParam.Value()
 			}
-
+			question := param.Name()
+			// add a prompt to question name to indicate if the boolean param is required to be true
+			if param.ValueType() == "boolean" && param.Validation() == "^true$" {
```

Comment:
Since we have a regexp validation, can you add an interactive validator? Basically this:
```
if param.ValueType() == "boolean" && param.Validation() == "^true$" {
  truthValidator := interactive.RegExp(param.Validation())
}
input := interactive.Input{
  Question: question,
  Help:     fmt.Sprintf("%s: %s", param.ID(), param.Description()),
  Required: param.Required(),
  Validators: []interactive.Validator{
    truthValidator,
  },
}
```
Or something like that... this way the prompt will only let the user through if they select `true`